### DISCUSSION TITLE
Comments

### DISCUSSION THREAD

**Discussion by Xhanti** [thegeeklab/hugo-geekblog#150] [Q&A] [answered]
Hi, 
Are they any plans to add a comments section to this project? 
Very good blog theme I'm enjoying it so far.

Regards

**Comment by xoxys** [comment, marked as answer]
Hi, thanks. We don't plan to integrate any kind of commenting system, sorry.

**Reply by Xhanti** [reply]
Thank you for the feedback :)

**Reply by xoxys** [reply]
You are welcome :) You might want to look into this topic https://gohugo.io/content-management/comments/. As Hugo supports to overload files from themes, integration it on your own might be possible without that much work. 

Disqus for example just need to be enabled in the config, and you can add the internal template by overwriting this template https://github.com/thegeeklab/hugo-geekblog/blob/main/layouts/_default/single.html

**Reply by Xhanti** [reply]
Thank you for the suggestion. I am really enjoying your theme, very nice and simple
